index.js

```diff
@@ -14,5 +14,5 @@ function isBinary(buf) {
 }
 
 module.exports = function (buf) {
-	return !isBinary(buf) && /<svg[^>]*>/.test(buf);
+	return !isBinary(buf) && /^<svg[^>]*>.*<\/svg>$/.test(buf);
```

Comment:
This won't work as SVG documents can start with an XML declaration. The end part is a good idea though. Also trying to parse it as XML.